src/server/pfs/server/driver_file.go

```diff
@@ -523,7 +524,7 @@ func (d *driver) getFileSet(ctx context.Context, commit *pfs.Commit) (*fileset.I
 			if baseCommitInfo.Finished == nil {
 				return nil, pfsserver.ErrBaseCommitNotFinished{
 					BaseCommit: baseCommit,
-					Commit:     commit,
+					Commit:     commitInfo.Commit,
```

Comment:
Was this change made because the commit passed into this method may not be resolved? If so, I would generally lean towards internal methods like this being able to rely on the invariant that commits are resolved, but if there is a case where they are not then a TODO for this would be fine.